package-lock.json

```diff
@@ -10,6 +10,7 @@
       "dependencies": {
         "@heroicons/vue": "^2.0.11",
         "@pinia/testing": "^0.0.14",
+        "@types/color": "^3.0.3",
```

Comment:
btw, what is this package used for? I try to read its package's home page, but I couldn't get the information.